test/config.spec.ts

```diff
@@ -56,15 +56,20 @@ describe('getStampsConfig', () => {
   const POSTAGE_TTL_MIN = '200'
   const POSTAGE_REFRESH_PERIOD = '10'
 
-  const values: { env: EnvironmentVariables; output: StampsConfig | undefined }[] = [
+  const values: { env: EnvironmentVariables; output: StampsConfig | undefined; description: string }[] = [
     // Postage stamps are disabld
-    { env: {}, output: undefined },
+    { env: {}, output: undefined, description: 'undefined for no input' },
     // Hardcoded postage stamp
-    { env: { POSTAGE_STAMP }, output: { mode: 'hardcoded', stamp: POSTAGE_STAMP } },
+    {
+      env: { POSTAGE_STAMP },
+      output: { mode: 'hardcoded', stamp: POSTAGE_STAMP },
+      description: '{mode: hardcoded, stamp} for {POSTAGE_STAMP}',
+    },
     // Shoudl return hardcoded postage stamp even if autobuy values are provided
```

Comment:
I mean since you have the `description` field maybe to comments are not necessary?